Snakefile

```diff
@@ -983,7 +997,8 @@ rule trinity_DE:
     1
   shell:
     """
-    num_replicates=`awk '{{print $2}}' {input.samples} | sort | uniq | wc -l` &> {log}
+    conda list | grep bioconductor-edger > {log} # prints version to log
```

Comment:
I think I would prefer `Rscript -e 'packageVersion("edgeR")'` here, instead of listing all conda packages